ietf-optical-impairment-topology.yang

```diff
@@ -605,42 +635,316 @@ module ietf-optical-impairment-topology {
+           This parameter corresponds to the worst case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required."; 
+      }
+      leaf roadm-pmax {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the best case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required.";        
+      }
+      leaf roadm-ptyp {
+        type decimal64 {
+          fraction-digits 2;
+        }
+        units dBm ;
+        description 
+          "If the drop path has additional loss that is added, for example, to hit target power levels into a 
+           drop path amplifier,or simply,to reduce the power of a “strong” carrier(due to ripple,for example), 
+           then the use of the ROADM input power levels and the above drop losses is not appropriate.
+           This parameter corresponds to the typical case per carrier power levels expected 
+           at the output of the drop block.To be noted that the net power levels from the drop block consider 
+           these power levels,and the power levels calculated with the drop loss attributes,if both are defined.
+           This comparison  captures the case if the ROADM input power levels are low enough such that extra
+           path loss is not required.";        
+      }
+      leaf roadm-osnr {
+        type decimal64 {
+          fraction-digits 5;
+        }
+        units "dB"; 
+        description 
+          "Optical Signal-to-Noise Ratio (OSNR).
+           Expected OSNR contribution of the drop path amplifier(if present) for the case of additional 
+           drop path loss (before this amplifier) in order to hit a target power level (per carrier).
+           If both, the OSNR based on the ROADM input power level(Pcarrier=Pref+10Log(carrier_baudrate/ref_baud)+delta_power)

```

Comment:
long line: add spaces around operators ' + '
change worst to minimum value

Reply:
ok